api/test/integration/auth.e2e-spec.ts

```diff
@@ -126,6 +126,50 @@ describe('Auth Controller Tests', () => {
     expect(loggedInUser.activeRefreshToken).not.toBeNull();
   });
 
+  it('should login successfully as user who has not accepted terms but now is', async () => {
```

Comment:
The phrasing of this it block I'm just find to be a lil confusing, is it maybe missing the end?

Reply:
I will try to rephrase, intention is that the user is now accepting